src/main/java/uk/nhs/hee/tis/trainee/forms/repository/LtftFormRepository.java

```diff
@@ -37,6 +37,14 @@
 @Repository
 public interface LtftFormRepository extends MongoRepository<LtftForm, UUID> {
 
+  /**
+   * Count the number of LTFT forms belonging to the given trainee.
```

Comment:
That's a pretty generic description of the query :>

Reply:
Red-Green-Refactor-ReJavadoc? 😄
I've added "non-draft" to the description.